### ISSUE TITLE
If there is no data returned for a report the callback method is not fired

### ISSUE THREAD

**Issue by shoecake** [MountainWarehouse/EasyMWS#67]
If i request a report (e.g. _GET_FLAT_FILE_ORDERS_DATA_ ) and the report returns no data (i.e. ReportProcessingStatus in ReportRequestInfo is _DONE_NO_DATA_ ) then the call back method is not called.

This is an issue for me as I still need the method being called so I can process the fact no data was available.

Is there an option/flag I can set in the QueueReport method to specify the callback should still be run?

**Comment by uber-rob** [comment]
I don't believe this is something we've coded for - yet! My thinking is that the callback method should just be called but with a null data payload - but perhaps that's confusing... Will discuss w/ team this week or if this is urgent for you just send a pull request.

**Comment by cazangabriel** [comment]
Interesting, I didn't think about this scenario.. This is more interesting than playing #HeroesOfTheStorm! so I took a short break from it and created a pull request for the feature, but I do have some doubts it will be merged this weekend.. anyway - I can sleep with a clear conscience now :)

Enjoy the new functionality!
